index.html

```diff
@@ -1793,22 +1793,22 @@ <h3>Matthew Clark</h3>
     <div class="shantanu bio">
         <h3>Shantanu Kadam</h3>
         <div class="line"></div>
-        Hey there! I'm a physics major who also appreciates math (I know you didn't see that coming) and cares about the environment. When I'm not at the chalkboard, you can find me swing dancing, cooking, daydreaming about Siberian Huskies, exploring Berkeley for vegan treats, perpetually drinking tea, reading, and (most importantly!) sleeping. Home sweet home is back in Alabama, so if you have any concerns about being far from home, please share them with me! ROHP led to my first friendships at Cal, and I can't wait to share this Cal-tastic experience with you :) Please reach out if you'd like to talk about Physics, Math, EEP (Environmental & Economic Policy), succeeding in physics, getting research, going out-of-state for college, being vegan in Berkeley, deCals, 7-breadth requirements, Society of Physics Students, RCSA, ROHP, the benefits of being a Regents' Scholar, planning your first semester, on-campus housing, concerns about Berkeley. 
+        Hey there! I'm a physics major who also appreciates math (I know you didn't see that coming) and cares about the environment. When I'm not at the chalkboard, you can find me swing dancing, cooking, daydreaming about Siberian Huskies, exploring Berkeley for vegan treats, perpetually drinking tea, reading, and (most importantly!) sleeping. Home sweet home is back in Alabama, so if you have any concerns about being far from home, please share them with me! ROHP led to my first friendships at Cal, and I can't wait to share this Cal-tastic experience with you :) Please reach out if you'd like to talk about Physics, Math, EEP (Environmental & Economic Policy), succeeding in physics, getting research, going out-of-state for college, being vegan in Berkeley, deCals, 7-breadth requirements, Society of Physics Students, RCSA, ROHP, the benefits of being a Regents' Scholar, planning your first semester, on-campus housing, concerns about Berkeley.
```

Comment:
This doesn't look like it's your fault, but see if you can Shantanu's bio to be formatted like everyone else's (i.e. multiple lines). Same thing if there's any other bios that for some reason are only one line.